flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java

```diff
@@ -218,6 +218,17 @@
 		.withDescription("Defines the directory where the Flink logs are saved. It has to be an absolute path." +
 			" (Defaults to the log directory under Flink’s home)");
 
+	/**
+	 * This options is here only for documentation generation, it is only
```

Comment:
It would be good to explicitly link to `config.sh#KEY_ENV_PID_DIR `and `config.sh#DEFAULT_ENV_PID_DIR`.

Reply:
I cannot link the config.sh file  in the java document with {@link}. Can you help me ?

Reply:
You can just refer to the files as "bin/config.sh"; that's sufficient.

Reply:
Thanks a lot. Please take a look.